### PR TITLE
feature-erms-2603

### PR DESCRIPTION
**Pull request by croqueGrec09**
as of ERMS-2603, the deletion of users has been completed and is ready for testing. Along with that, residuals of Doc.creator and Doc.user has been deleted as well

### CODE REVIEW

File: app/grails-app/controllers/de/laser/SubscriptionController.groovy [hunk 1]
```diff
@@ -3222,13 +3224,14 @@ class SubscriptionController
     }
 
     @DebugAnnotation(test = 'hasAffiliation("INST_USER")')
-    @Secured(closure = { ctx.springSecurityService.getCurrentUser()?.hasAffiliation("INST_USER") })
+    @Secured(closure = { ctx.springSecur
+        ityService.getCurrentUser()?.hasAffiliation("INST_USER") })
```
Comment (Moetez16) [inline app/grails-app/controllers/de/laser/SubscriptionController.groovy:3228]: ctx.springSecur?

File: app/grails-app/controllers/de/laser/SubscriptionController.groovy [hunk 2]
```diff
@@ -3222,13 +3224,14 @@ class SubscriptionController
...
     def deleteDocuments() {
         def ctxlist = []
 
         log.debug("deleteDocuments ${params}");
 
-        docstoreService.unifiedDeleteDocuments(params)
+        docstoreService.unifiedDeleteDocuments(params)CHANGELOG
```
Comment (Moetez16) [inline app/grails-app/controllers/de/laser/SubscriptionController.groovy:3234]: CHANGELOG?

Comment (croqueGrec09) [reply app/grails-app/controllers/de/laser/SubscriptionController.groovy:3234]: IntelliJ was too slow ...

Comment (croqueGrec09) [reply app/grails-app/controllers/de/laser/SubscriptionController.groovy]: Resolved in 4fc43af

File: app/grails-app/controllers/de/laser/SubscriptionController.groovy (hunk 1 above)
Comment (croqueGrec09) [reply app/grails-app/controllers/de/laser/SubscriptionController.groovy]: IntelliJ read my entry too late, fixed in 4fc43af

File: app/grails-app/views/templates/documents/_card.gsp [hunk 3]
```diff
@@ -119,7 +114,7 @@
                                 </div>
                             </g:else>
                             <%-- START Third Button --%>
-                            <g:if test="${!(ownobj instanceof Org) && ownobj?.showUIShareButton() && accessService.checkMinUserOrgRole(user,docctx.owner.owner,"INST_EDITOR")}">
+                            <g:if test="${!(ownobj instanceof Org) && ownobj?.showUIShareButton() && editable}">
```
Comment (Moetez16) [inline app/grails-app/views/templates/documents/_card.gsp:117]: Ne. Bitte so lassen. Ich glaube ich habe so eingebaut wegen Umfragen!

Comment (croqueGrec09) [reply app/grails-app/views/templates/documents/_card.gsp:117]: Wo definierst du user?

Comment (croqueGrec09) [reply app/grails-app/views/templates/documents/_card.gsp]: Den Bug konnte ich nicht mehr reproduzieren, behoben in aaeb7fa

**Review by Moetez16 — APPROVED** [review]

**Pull request merged**